modules/ROOT/pages/indexes/syntax.adoc

```diff
@@ -113,7 +113,6 @@ ON (r.propertyName_1)
 [OPTIONS “{“ option: value[, …] “}”]
 ----
 
-Point indexes have only one index provider available, `point-1.0`.
```

Comment:
add a comment like for range about not having settings and then remove options from the syntax

Reply:
Well I left the text index ones since both providers still exist :shrug: . I did remove any mention of the index-provider option being used to select one though

I wasn't sure about removing the options from the syntax since it is still supported to supply options, just not recommended and we will ignore it. But I'd be happy to see them go if you're saying they don't have to stay

Reply:
I think removing it altogether is better.